src/conn/conn_api.c

```diff
@@ -2785,8 +2785,8 @@ wiredtiger_open(const char *home, WT_EVENT_HANDLER *event_handler, const char *c
       __conn_load_extension, __conn_add_data_source, __conn_add_collator, __conn_add_compressor,
       __conn_add_encryptor, __conn_add_extractor, __conn_set_file_system, __conn_add_storage_source,
       __conn_get_storage_source, __conn_get_extension_api};
-    static const WT_NAME_FLAG file_types[] = {{"checkpoint", WT_DIRECT_IO_CHECKPOINT},
-      {"data", WT_DIRECT_IO_DATA}, {"log", WT_DIRECT_IO_LOG}, {NULL, 0}};
+    static const WT_NAME_FLAG file_types[] = {
+      {"data", WT_DATA_FILE_TYPE}, {"log", WT_LOG_FILE_TYPE}, {NULL, 0}};
```

Comment:
Question: Why is "checkpoint" no longer in the list?
This is not a review comment. It is something I do not understand.

Reply:
Hi Tod, checkpoint data type was only used for direct_io, whereas the other are useful datatypes we use for other configurations.